examples/model_compress/README.md

```diff
@@ -0,0 +1,47 @@
+# Run model compression examples
+
+You can run these examples easily like this, take torch pruning for example
+
+```bash
+python main_torch_pruner.py
+```
+
+Model compression can be configured in 2 ways
```

Comment:
This example uses AGP Pruner. Initiating a pruner needs a user provided configuration which can be provided in two ways: